docs/site/tutorials/todo-list/todo-list-tutorial-repository.md

```diff
@@ -156,31 +126,13 @@ Let's do the same on the `TodoRepository`:
 {% include code-caption.html content="src/repositories/todo.repository.ts" %}
```

Comment:
apologize that I wasn't following the implementation of this feature very closely.  Do you mind explaining why we need to register for `TodoRepository`?  is it for the `belongsTo` route?

Reply:
Yes! I'll put a comment so it's more clear.